### ISSUE TITLE
sun_color_r,g,b not configurable by map config file

### ISSUE THREAD

**Issue by PalmliX** [NVIDIA/Q2RTX#208]
sun_color_r,g,b settings reset on map load, even when defined in a map config file. It would be nice for custom content to be able to set sun color via map config. The ideal behavior would reset to physical_sky defaults when not defined in map config so as to not conflict with other content.

**Comment by Cannonaire** [comment]
I came here to report sun color settings reset on map load as well. I'll just add that an indicator/overlay for sun values, similar to the FPS overlay, would be useful. An in-game time of day indicator/overlay would also be useful because you can't tell when using 12x time mode.

**Comment by Paril** [comment]
The reason this happens is because of physical_sky's preset stuff basically stomping over all of the sun variables on map load. I'm not sure how you'd resolve this easily with the way the sky system is setup currently.

**Comment by WabbitToonz** [comment]
in the map ,cfg  use the command wait
wait 10 as the first line  and  every time you enter that map the sun color will be as you set it

**Comment by PalmliX** [comment]
Thanks for the tip @WabbitToonz! However it didn't seem to work, in fact it seemed to negate the rest of the map cfg, did I do this right?
![image](https://user-images.githubusercontent.com/7981637/170822904-7b3d5d26-d039-4de6-aa12-04d75e2cf5d4.png)

**Comment by WabbitToonz** [comment]
i am using this on  version 160 .  map name base1.cfg 

wait 10
sun_color_b "0.270000"
sun_color_g "0.290000"
sun_color_r "1.450000"
every time  i go to base 1 it sticks

**Comment by PalmliX** [comment]
Ah I figured it out, putting wait at the beginning seemed to interfere with all the various settings after it, so I moved it to the end leaving only the sun_color settings after it and it works now. Thanks again @WabbitToonz!

![image](https://user-images.githubusercontent.com/7981637/170866018-fc604e82-ac9e-450c-9a84-1d923214032d.png)

Guess I'll close the issue since this DOES make sun_color configurable by map config now, albeit with a slight delay after loading. That's good enough for me.
